server/tests/batch_comparison/snapshots/snap_test_batch_inventory.py

```diff
@@ -156,3 +156,21 @@
 Tabulator 2 - BATCH1,2,1,3,2,1\r
 Tabulator 2 - BATCH2,2,0,6,2,4\r
 """
+
+snapshots[
+    "test_batch_inventory_hart_cvr_upload 1"
+] = """Batch Name,Number of Ballots\r
+BATCH1,3\r
+BATCH2,3\r
+BATCH3,2\r
+BATCH4,1\r
+"""
+
+snapshots[
+    "test_batch_inventory_hart_cvr_upload 2"
+] = """Batch Name,Choice 1-1,Choice 1-2\r
+BATCH1,1,2\r
+BATCH2,2,1\r
+BATCH3,0,0\r
+BATCH4,0,0\r
```

Comment:
It seems like BATCH4 has votes for both Choice 1-1 and Choice 1-2 based on this line from the tests:

```py
build_hart_cvr("BATCH4", "1", "1-4-1", "1,1,0,0,1", add_write_in=True)
```

Do you know why they aren't showing up here? If it's because it's an overvote, I can't figure out where that's getting filtered out.

Also, how are write-ins handled?

Reply:
Its getting filtered because of both being write-ins, going to update the test a bit to make things clearer (and ask about what to do with write-ins in slack)